### PR TITLE
stable/pomerium: add member to owners

### PR DESCRIPTION
**Pull request by desimone**
Signed-off-by: Bobby DeSimone <bobbydesimone@gmail.com>

### CODE REVIEW

File: stable/pomerium/OWNERS [hunk 1]
```diff
@@ -1,5 +1,7 @@
 approvers:
 - desimone
+- travisgroth
 reviewers:
 - desimone
-- victornoel
\ No newline at end of file
+- victornoel
+- travisgroth
```
Comment (maorfr) [inline stable/pomerium/OWNERS]: missing a newline?
and please update the maintainers section in Chart.yaml :)

Comment (desimone) [reply stable/pomerium/OWNERS]: Done 👍

**Comment by desimone** [comment]
@maorfr should be good to go 👍

**Comment by maorfr** [comment]
/lgtm

thanks!

**Pull request merged**